nucleus/scene.py

```diff
@@ -38,16 +42,15 @@ class Frame:
     """
 
     def __init__(self, **kwargs):
-        self.items = {}
+        self.items: Dict[str, DatasetItem] = {}
         for key, value in kwargs.items():
-            self.items[key] = value
-
-    def __post_init__(self):
```

Comment:
__post_init__ does not work on default classes.

Reply:
nice catch!!